client/signup/jetpack-connect/index.jsx

```diff
@@ -68,14 +81,29 @@ const JetpackConnectMain = React.createClass( {
 	},
 
 	onURLEnter() {
+		this.props.recordTracksEvent( 'calypso_jpc_url_submit', {
+			jetpack_funnel: this.state.currentUrl,
```

Comment:
@johnHackworth: yes, that's a different thing. I will elaborate in the post. (`jetpack_funnel` is a database table elsewhere.)